pythonx/netranger/netranger.py

```diff
@@ -185,11 +185,17 @@ def __init__(self, fullpath, name, level=0, buf=None):
     def re_stat(self, lazy=False):
         self.linkto = None
         if os.path.islink(self.fullpath):
-            self.linkto = os.readlink(self.fullpath)
-
+            try:
+                self.linkto = os.readlink(self.fullpath)
+            except PermissionError:  # added eyal
+                self.stat = None
```

Comment:
I mean by modifying this line:
```python
self.stat = None
Vim.log(f'Permission denied: {self.name}')
```

Reply:
Sorry for a bit of misunderstanding on my part. I think that echoing "permission error" would be OK, while also written the full error to the log. But the important thing is that it should continue and list the dir contents, even if one member is problematic.